bedrock/firefox/forms.py

```diff
@@ -42,7 +42,7 @@ def clean(self, value):
 
 
 class SendToDeviceWidgetForm(forms.Form):
-    number = USPhoneNumberField(required=False)
```

Comment:
Doesn't look like it's used any more - should we keep the `USPhoneNumberField` class around for possible future use or drop it to clean the code base?

Reply:
I think you're right. It might be useful in future, but that's why we have git history right :)